lib/resources/crontab.rb

```diff
@@ -73,7 +77,7 @@ def parse_crontab_line(l)
     end
 
     def crontab_cmd
-      @user.nil? ? 'crontab -l' : "crontab -l -u #{@user}"
+      @destination.nil? ? 'crontab -l' : "crontab -l -u #{@destination}"
```

Comment:
I don't think this works... `-u` specifically takes a user. On a stock Ubuntu 16.04 box, which has a `/etc/cron.d/mdadm` file, `crontab -l -u /etc/cron.d/mdadm` fails. I think you just need to read in the file.